examples/daal/cpp_sycl/source/k_nearest_neighbors/bf_knn_dense_regression_batch.cpp

```diff
@@ -0,0 +1,185 @@
+/* file: bf_knn_dense_regression_batch.cpp */
+/*******************************************************************************
+* Copyright 2014-2020 Intel Corporation
```

Comment:
For new files, we must indicate the year of their appearance - not the birth of the library.

```suggestion
* Copyright 2020 Intel Corporation
```